src/DGtal/shapes/doc/moduleMesh.dox

```diff
@@ -0,0 +1,144 @@
+/**
+* @file moduleMesh.dox
+*
+* @author Bertrand Kerautret (\c kerautre@loria.fr )
+* LORIA (CNRS, UMR 7503), University of Nancy, France
+
+* @date 2014/06/27
+*
+* Documentation file for feature moduleMesh
+*
+* This file is part of the DGtal library.
+*/
+
+/* 
+* Useful to avoid writing DGtal:: in front of every class.
+* Do not forget to add an entry in src/DGtal/base/Config.h.in !
+*/
+namespace DGtal {
+//----------------------------------------
+/*!
+@page moduleMesh Manipulating Simple Mesh 
+@author Bertrand Kerautret
+
+[TOC]
+
+\section sectmoduleMesh1 Simple Mesh Class 
+
+
+The aim of this class is to handle basic simple mesh mainly for the
+task of import/export/display of digital surface. This class is able
+to deal with triangulated, quad or polygonal faces and can include
+specific colors to display faces. It also provides several static
+tools to generate simple shapes like tubular objects or the 3d surface
+from height map.
+
+The mesh object store explicitly each vertex and each face are represented with the list of point index.       
```

Comment:
stores explicitly the vertices and each face is represented as a list of vertex indices